index.html

```diff
@@ -7,5 +7,6 @@
 <title>Welcome to Potionfy</title>
 </head>
 <body>
+  <h1>Welcome to los Angels ! Enjoy your stay</h1>                                                                 
```

Comment:
Could you please remove 'Enjoy your stay' String and mark Welcome to Los Angels in red color?